stats/registry_test.go

```diff
@@ -0,0 +1,50 @@
+package stats_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/influxdb/stats"
+)
+
+func TestEmptyStatistics(t *testing.T) {
+	found := stats.Collect(stats.Root.Open(), true)
+
+	if length := len(found); length != 0 {
+		t.Fatalf("non empty initial state. got %d, expected: %d", length, 0)
+	}
+}
+
+// Test that we can create one statistic and that it disappears after it is deleted twice.
+func TestOneStatistic(t *testing.T) {
+
```

Comment:
There is no need to nil out a local variable, the lifetime of a variable
does not live beyond it's last use.

On Tue, 1 Mar 2016, 06:51 Jon Seymour, <notifications@github.com> wrote:

> In stats/registry_test.go
> <https://github.com/influxdata/influxdb/pull/5830#discussion_r54466491>:
>
> > +	"testing"
> > +
> > +	"github.com/influxdata/influxdb/stats"
> > +)
> > +
> > +func TestEmptyStatistics(t *testing.T) {
> > +	found := stats.Collect(stats.Root.Open(), true)
> > +
> > +	if length := len(found); length != 0 {
> > +		t.Fatalf("non empty initial state. got %d, expected: %d", length, 0)
> > +	}
> > +}
> > +
> > +// Test that we can create one statistic and that it disappears after it is deleted twice.
> > +func TestOneStatistic(t *testing.T) {
> > +
>
> Will fix this with a subsequent push.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdata/influxdb/pull/5830/files#r54466491>.
>